doc/how_to_guides/offline.md

```diff
@@ -16,8 +16,13 @@ The way of transfer depends on the environment. The files to be transferred are
 ## Import the vulnerability database file on the offline system
 The vulnerability database should be copied into ~/.cache/cve-bin-tool
 
-## Run cve-bin-tool with --update never option
-In an offline environment, when running a scan specify `--update never` so that cve-bin-tool doesn't attempt to download the latest database files.
+## Create configuration file
+A configuration file is used to specify default behaviour for the cve-bin-tool. When running in an offline environment the `disable_version_check` option should be set to `false` to prevent the tool checking for a newer version of the tool.
```

Comment:
Wait, should `disable_version_check` be `false` or `true`?  I'd have assumed that `disable_version_check` would be `true` to disable the version check.  Is the logic here backwards in the doc or the tool?